iron-component-page.html

```diff
@@ -643,8 +643,8 @@
 
         _analyzerChanged: function() {
           var analyzer = this._analyzer;
-          this._setDocElements(analyzer && analyzer.elements ? analyzer.elements : []);
-          this._setDocBehaviors(analyzer && analyzer.behaviors ? analyzer.behaviors : []);
+          this._setDocElements(analyzer ? analyzer.elementsForFolder(this.base) : []);
+          this._setDocBehaviors(analyzer ? analyzer.elementsForFolder(this.base) : []);
```

Comment:
Returns everything under a subpath shared by the specified element.

elementsForFolder('components/paper-button/paper-button.html') will return everything under 'components/paper-button'